packages/core/src/browser/view-container.ts

```diff
@@ -172,6 +179,19 @@ export class ViewContainer extends BaseWidget implements StatefulWidget, Applica
         if (this.options.progressLocationId) {
             this.toDispose.push(this.progressBarFactory({ container: this.node, insertMode: 'prepend', locationId: this.options.progressLocationId }));
         }
+
+        this.getSashHoverDelay();
```

Comment:
it might be a good idea to wait for the `corePreferences.ready` promise before attempting to retrieve these preferences on start

Reply:
@kenneth-marut-work Thanks for the feedback! I have pushed changes that addresses your comments.